cli/main.c

```diff
@@ -148,12 +150,14 @@ static int gui(int argc, char **argv)
 		 "gui refresh period in seconds (default: 1 second)"},
 		{"duration", 'd', "", CFG_INT, &cfg.duration, required_argument,
 		 "gui duration in seconds (-1 forever)"},
+		{"bw_type", 'b', "NUM", CFG_POSITIVE, &cfg.bw_type,
+		 required_argument, "gui bw type: 0 raw(default), 1 payload"},
```

Comment:
Please use a CFG_CHOICES for the options or make a different option for each type (--payload, --raw). 0 and 1 are completely unintuitive.